packages/jest-get-type/src/index.ts

```diff
@@ -57,7 +57,7 @@ export function getType(value: unknown): ValueType {
     return 'symbol';
   }
 
-  throw new Error(`value of unknown type: ${value}`);
+  throw new Error(`value of unknown type: ${typeof value} (${value})`);
```

Comment:
```suggestion
  throw new Error(`value of unknown type: ${value} (typeof ${typeof value})`);
```

I think that makes more sense in case `typeof` returns something not super useful (and keeps it closer to what we have today)

Reply:
I don't necessarily agree that it's better that way. The error is about the type, so I would intuitively expect the thing after the colon to be the type in question. If the typeof return isn't very useful, only in those edge cases would the value become the primary information, so it shouldn't come first in most cases.